### DISCUSSION TITLE
How to Filter Rows with Null Values in Milvus?

### DISCUSSION THREAD

**Discussion by ducanh997** [milvus-io/milvus#34327] [Q&A and General discussion]
I have a dynamic field in a collection in Milvus, and I need to filter rows where the value is null. How can I achieve this in Milvus?

Thanks for your support!

**Comment by yhmo** [comment]
So far as I know, milvus doesn't support null values.

**Comment by xiaofan-luan** [comment]
@smellthemoon please help on explain it

**Comment by smellthemoon** [comment]
@ducanh997 hello, null support is under development. It will has 'nullable' in fieldSchema, which means you can insert null in the field. The nullable of a dynamic column must be true. If you want to filter rows in dynamic, you can use "is null" expression when query. The null function is planned to be launched in 2.5 version.
